<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobAfterEffectsSettingsWidgetUI</class>
 <widget class="QGroupBox" name="JobAfterEffectsSettingsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>After Effects Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mVerticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="1">
      <widget class="FieldLineEdit" name="mCompEdit">
       <property name="field" stdset="0">
        <string>comp</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel8">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="FieldSpinBox" name="mWidthSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>width</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Comp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel9">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="FieldSpinBox" name="mHeightSpin">
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="field" stdset="0">
        <string>height</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>221</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fieldlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FieldSpinBox</class>
   <extends>QSpinBox</extends>
   <header>fieldspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
